src/values/ObjectValue.js

```diff
@@ -244,7 +247,15 @@ export default class ObjectValue extends ConcreteValue {
   // to return AbstractValue for unknown properties.
   _isSimple: BooleanValue;
 
-  isTemplate: void | true;
+  // If true, then unknown properties should return transitively simple abstract object values
+  _simplicityIsTransitive: BooleanValue;
```

Comment:
`| AbstractValue` --- let's do this new one right...

Reply:
Sure, but in another pull request.